new-frontiers-14/frontier-station-14
# Cryogenics Overhaul

adding cryo chems to work after death
balancing cryo chem formulas based on objectives
balancing cryo chems healing based on formulas
making cryo pods more efficient
modifying how the body metabolize chemical mixes
adding a way to split solution reagents evenly
renaming Tramoxadone to Traumoxadone




## About the PR

added new cryo chem stelloxadone

modified dexo cryo chem to be a more complex solution to fall in line with other cryo chems but also improving its capabilities

modified cryox to not be an omni heal and negate the point of cryo chems but added blood restoration and significantly improved its asphyxiation  healing

modified alox cryo chem to heal cold damage in addition to other burn damages so that omnizine was no longer the only way to heal dead cold damage

all cryo chems now work on the dead

tramoxadone is now traumoxadone

cryo pod is more efficient at administering chemicals

the body can now metabolize multiple meds at once,  still preventing from poison power gaming (consider adding more medical mixes that result in dangerous chemicals like razorium to prevent medical power gaming)


## Why / Balance

this is in my effort to help improve medical over all and making cryo a more impactful way of healing to make it actually used in game rather than being a joke concept.  

## How to test

- created a cryo setup
- smashed urist with throngler
- ensure that a beaker with stellox, traumox, alox, and dex-ox, and cryox, would all be injected into the body at .25 units each note that this should also be the same even if the units in solution are not balanced
- observe that the body has .5 of each injected into the body and they they are all being metabolized as none are poisons
- change solution to test each chem individually to ensure they heal as expected note that stell heals both poison and radiation.
- tested that cryo heals blood loss but not other damages and after death
- tested that dex-ox heals genetic more effectively and after death 
- tested that alox heals cold
- tested that all of traumox's tests are accurate
- tossed 5 different chems into a beaker
- tested only 3 chems would process at a time randomly chosen
- repeated this test with medicals and narcotics also mixed with the poisons and ensured that everything was injected except poisons, max 3 poisons were injected into the body

## Media


- [x] I have added screenshots/videos to this PR showcasing its changes in game, **or** this PR does not require an in game showcase

## Breaking changes


**Changelog**


:cl:
- add: new cryo chems
- tweek: cryo chems to work after death
- tweek cryo tube more efficient
